modules/s3-bucket/main.tf

```diff
@@ -18,14 +18,6 @@ locals {
 
 data "aws_partition" "current" {}
 
-data "template_file" "bucket_policy" {
```

Comment:
No need to remove. We provide a arm binary. 

https://github.com/cloudposse/terraform-provider-template

https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-sns-alarms/pull/34/files#diff-202419d570b64c1010bf146c14f4a9fa017a98b9c936d9af293c7802a8f79e98R9

Reply:
Gotcha will do